src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private static final String name = "rectangle";
+    private int firstLeg;
+    private int secondLeg;
+    private double area;
+
+    public Rectangle(int firstLeg, int secondLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        setColor(color);
+        setArea(getArea());

```

Comment:
```suggestion
        setColor(color);
        setArea(getArea());

```

you don't need use setters in the same class

Reply:
I don't quite get it here, can you be more specific?